independent-projects/qute/core/src/main/java/io/quarkus/qute/ValueResolvers.java

```diff
@@ -521,4 +523,13 @@ private static CompletionStage<Object> mapResolveAsync(EvalContext context) {
         }
     }
 
+    @SuppressWarnings({ "rawtypes" })
+    private static boolean checkAllValuesEmpty(Map map) {
+        for (Object object : map.values()) {
+            Collection collection = (Collection) object;
+            if (!collection.isEmpty())
+                return false;
+        }
+        return true;
+    }
```

Comment:
I think introducing changes to the `qute` templating engine is way out of scope for this PR. Please remove these changes.

If you introduce proper data classes to represent persistence units as I suggested above, you will be able to simply implement `allValuesEmpty` as an instance method in one of the data classes.